src/natcap/invest/finfish_aquaculture/finfish_aquaculture.py

```diff
@@ -79,31 +79,31 @@
             "required": True,
         },
         "use_uncertainty": {
-            "name": "Enable uncertainty analysis",
+            "name": "do uncertainty analysis",
             "about": "Enable uncertainty analysis.",
             "type": "boolean",
             "required": True,
         },
         "g_param_a_sd": {
-            "name": "Standard Deviation for Parameter (a)",
+            "name": "α standard deviation",
             "about": (
-                "Standard deviation for fish growth parameter a. "
+                "Standard deviation for fish growth parameter α. "
                 "This indicates the level of uncertainty in the "
-                "estimate for parameter a."),
+                "estimate for parameter α."),
             "type": "number",
             "required": "use_uncertainty",
         },
         "g_param_b_sd": {
-            "name": "Standard Deviation for Parameter (b)",
+            "name": "β standard deviation",
             "about": (
-                "Standard deviation for fish growth parameter b. "
+                "Standard deviation for fish growth parameter β. "
                 "This indicates the level of uncertainty in the "
-                "estimate for parameter b."),
+                "estimate for parameter β."),
             "type": "number",
             "required": "use_uncertainty",
         },
         "num_monte_carlo_runs": {
-            "name": "Number of Monte Carlo Simulation Runs",
+            "name": "number of Monte Carlo runs",
```

Comment:
Yeah I'm not quite sure about this one... "Ricker" and "Beverton-Holt" are clearly names and so should be capitalized. I don't know if a "Monte Carlo simulation" is named after someone, but as a sort-of algorithm title, the capitalization looks right to me. I was thinking that these should be capitalized here in the `ARGS_SPEC` because they should be capitalized regardless of any capitalization rule we might apply later (e.g. capitalize the first letter of every word). You're right it should be documented somewhere, perhaps [here in the wiki](https://github.com/natcap/invest/wiki/spec_invest#the-args_spec)?

Reply:
> I  don't know if a "Monte Carlo simulation" is named after someone, but as a sort-of algorithm title, the capitalization looks right to me.

Yep, looks like the [Monte Carlo method's name is taken from the _Monte Carlo Casino_ in Monaco](https://en.wikipedia.org/wiki/Monte_Carlo_method), so definitely a name.  Let's keep the capitalization!

> I was thinking that these should be capitalized here in the ARGS_SPEC because they should be capitalized regardless of any capitalization rule we might apply later (e.g. capitalize the first letter of every word)

👍 

> You're right it should be documented somewhere, perhaps here in the wiki?

Yep!  That makes sense to me.